docs-v2/pages/connect/components.mdx

```diff
@@ -0,0 +1,501 @@
+import { Steps, Tabs } from 'nextra/components'
+import ArcadeEmbed from '@/components/ArcadeEmbed'
+import Callout from '@/components/Callout'
+import Image from 'next/image'
+
+# Running Components for Your End Users
+
+Pipedream Connect provides you with the ability to pre-configure, deploy and
+invoke [components](/components) **on behalf of [your end
+users](/connect/api#external-users)**.
+
+## What are Components?
```

Comment:
Same sentence-case suggestion here — I won't add comments for the remainder but if you can update the rest of the headings that would help